README.md

```diff
@@ -164,7 +164,7 @@ After deploying the binary to the board, the Nucleo's `LD1` LED will flash red a
 
 ## Serial Communication
 
-The boards can be communicated with through the serial interface exposed through the debug pins. You can use the USB-serial interface built into the Nucleo dev boards to communicate with the control boards by connecting the TX pin to the board's RX pin and the RX pin to the board's TX pin (transmit to recieve and vice versa). 
+The boards can be communicated with through the serial interface exposed through the debug pins. You can use the USB-serial interface built into the Nucleo dev boards to communicate with the control boards by connecting the TX pin to the board's RX pin and the RX pin to the board's TX pin (transmit to recieve and vice versa). Ensure the program running on the nucleo is not printing too.
```

Comment:
Why?

Reply:
this is something i noticed. i was trying to print from a pcb and route the data to a nucleo and then see it on a serial terminal window. it didnt work if the nucleo was also printing. kinda makes send too right. they're both trying to push their data through the same usb cable

Reply:
ohhhhh you mean a program is also running on the nucleo interfering with the serial. that makes sense since serial is not something thats like a bus. I'd maybe reword this to be clearer and maybe bold it lol. Ex. **Ensure that the Nucleo is not running an app that uses the serial pins (Ex. an app that uses `printf`).**